plugins/woocommerce/includes/admin/helper/class-wc-helper-updater.php

```diff
@@ -501,12 +520,9 @@ public static function get_updates_count() {
 	 * @return string Updates count markup, empty string if no updates avairable.
 	 */
 	public static function get_updates_count_html() {
-		$count = self::get_updates_count();
-		if ( ! $count ) {
-			return '';
-		}
-
+		$count      = self::get_updates_count_based_on_site_status();
 		$count_html = sprintf( '<span class="update-plugins count-%d"><span class="update-count">%d</span></span>', $count, number_format_i18n( $count ) );
+
```

Comment:
The bubble HTML is still present in the DOM when the number is count is `0`. It's not seen, as there is CSS that hides the element. Guess it's fine to leave as is, but just wanted to bring it to your attention in case it's not intended.

![image](https://github.com/woocommerce/woocommerce/assets/2803994/b07c787e-88a6-4a95-b4a7-ff257240163d)